app/controllers/users_controller.rb

```diff
@@ -9,18 +9,20 @@ def new
   end
 
   def index
-    @users = User.paginate page: params[:page],
+    @users = User.where(activated: true).paginate page: params[:page],
```

Comment:
`.where(activated: true)`
tìm hiểu về `scope in rails` và viết scope cho cái code trên hì